.github/workflows/PR-into-2022-2.yml

```diff
@@ -0,0 +1,53 @@
+name: Merge and PR into 2022.2
+
+run-name: Merge 2021.3 -> 2022.2
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - 'maintenance/mps20213'
+
+jobs:
+  merge-and-pr:
+    # only trigger if it was actually merged
+    if: github.event.pull_request.merged == true
+
+    runs-on: ubuntu-latest
+    steps:
+      # checkout the target branch
+      - name: Checkout 2022.2
+        uses: actions/checkout@v3
+        with:
+          ref: 'maintenance/mps20222'
+          fetch-depth: 0
+
+      # actually merge from the previous branch
+      - name: Perform merge from 2021.3 to 2022.2
+        run: |
+          # setup to allow to merge and commit
+          git config user.name 'github-actions[bot]'
+          git config user.email 'github-actions[bot]@users.noreply.github.com'
+          
+          # actual merge operation
+          git merge --no-ff --message "Merge 2021.3 -> 2022.2" origin/maintenance/mps20213 -- || ( echo "Merge failed. Please merge manually!" ; exit 1 )
+
+      # actually merge from the previous branch
+      - name: Prepare environment variables for PR
+        run: |
+          # strip the title of previous automatic PR content
+          NEW_TITLE='${{ github.event.pull_request.title }}'
+          NEW_TITLE="${NEW_TITLE// (2021.2 -> 2021.3)}"
```

Comment:
The title looks wrong.

Reply:
This part is actually only stripping the previous parentheses from the title (to be able to cascade these PRs without the title growing longer). I’ll add a comment to explain this.